WeatherApp/WeatherApp/Feature/Components/CustomNavBar.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+struct NavBar: View {
+
+    let font: Font = Font.custom("NDOT45inspiredbyNOTHING", size: 25)
+
+    var backAction: () -> Void
+
+    var body: some View {
+        HStack {
+            Button(action: backAction) {
+                Text("<")
```

Comment:
Also I would suggest using the chevron image in substitate for text.

I will create you an asset for this ;)

Reply:
I used text to get the look I was going for.